lesson01/assignment/scripts.js

```diff
@@ -29,5 +29,120 @@ document. addEventListener("DOMContentLoaded", function(){
     // Clear the form field so the user can try again
     inputField.value = ''
   })
+})
+
+document. addEventListener("DOMContentLoaded", function(){
```

Comment:
No need to add a second DOMContentLoaded listener function. All of the below could just be added to the event listener above.